tests/Visual/JUnit.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Pest\Logging\JUnit;
+use PHPUnit\Framework\AssertionFailedError;
+use PHPUnit\Framework\TestSuite;
+use PHPUnit\Framework\Warning;
+
+beforeEach(function () {
+    file_put_contents(__DIR__ . '/junit.html', '');
+});
+
+it('is can successfully call all public methods', function () {
+    $junit = new JUnit(__DIR__ . '/junit.html');
```

Comment:
Really nice with the tests. 

Should we also test the output? We can do that by having the junit.html file saved and checking if the result is identical with the generated one 🤔

Reply:
@olivernybroe I've added a test in `SingleTestOrDirectory` that runs using JUnit logging and asserts it matches the given snapshot.